Mesh_3/examples/Mesh_3/mesh_implicit_sphere_variable_size.cpp

```diff
@@ -54,8 +54,8 @@ int main()
   /// [Domain creation] (Warning: Sphere_3 constructor uses squared radius !)
   namespace p = CGAL::parameters;
   Mesh_domain domain = Mesh_domain::create_implicit_mesh_domain
-    (p::function = &sphere_function,
-     p::bounding_object = K::Sphere_3(CGAL::ORIGIN, 2.)
+    (sphere_function,
```

Comment:
can you comment on this change? it seems that named parameters are not used anymore

Reply:
If some parameters are mandatory, then only the `named_parameter` parameter should not have a default (there is some a  no default `CGAL_NP_TEMPLATE_PARAMETERS_NO_DEFAULT`). Then see my comment below with the `static_assert`. Note that one important point is to not break user code. So we should target to make all examples from the release `CGAL-5.4` to still compile with our changes.

Reply:
If you see some issue with full backward compatibility that does not seem simple to fix, you can collect them and we can discuss them.

Reply:
Any update on this topic?